code-examples/php/usage-examples/updateOne.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+require 'vendor/autoload.php';
+
+// Replace the uri string with your MongoDB deployment's connection string
+$uri = "mongodb://localhost:27017";
+
+$client = new MongoDB\Client($uri);
+
+// Get the "movies" collection in the "sample_mflix" database
+$collection = $client->sample_mflix->movies;
+
+try {
+    /* Update a document that has the title "Random Harvest" to have a
+    plot field with the specified value */
+    $result = $collection->updateOne(
+        [ 'title' => 'Random Harvest' ],
+        [ '$set' => [
+               'plot' => "A harvest of random numbers, such as: " . rand(),
```

Comment:
Better to use [`random_int()`](https://www.php.net/random_int) and specify a range of min/max values.